src/inference/src/dev/threading/cpu_streams_executor.cpp

```diff
@@ -21,6 +22,8 @@
 
 namespace ov {
 namespace threading {
+// maybe there are two CPUStreamsExecutors in the same thread.
+thread_local std::map<void*, std::shared_ptr<std::thread::id>> t_stream_count_map;
```

Comment:
hi @tiger100256-hu can you help explain why do we need this one?

Reply:
use std::shared_ptr<std::thread::id> to count  Imp:stream, when the thread is destroyed，the count of  std::shared_ptr<std::thread::id> will became 1, and local() can reuse  or release Imp:stream